libs/server-sdk-redis-source/src/redis_source.cpp

```diff
@@ -0,0 +1,75 @@
+#include <launchdarkly/server_side/integrations/redis/redis_source.hpp>
+
```

Comment:
Wondering if the clang diagnostic fails because it is somehow using cmake to build everything (and hence wouldn't know about adding `LD_BUILD_REDIS_SUPPORT`.)